src/App.tsx

```diff
@@ -2,72 +1,71 @@
+import React, { useMemo, useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
+
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+
+import { SearchInput } from './components/searchInput/searchInput';
+import Content from './components/content/content';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [delay, setDelay] = useState(300); // for change add setDelay
+
+  const [fullField, setFullField] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const [focus, setFocus] = useState(false);
+
+  const people = useMemo(() => {
+    return [...peopleFromServer];
+  }, []);
+
+  const peopleSearcher = useMemo(() => {
+    return people.filter(person => person.name.includes(fullField));
```

Comment:
The current filter logic does not handle the case where the input is spaces only. You should add a condition to skip filtering when `fullField.trim()` is an empty string.